genesyscloud/resource_genesyscloud_init_test.go

```diff
@@ -51,15 +51,14 @@ func (r *registerTestInstance) registerTestResources() {
 	providerResources[routingQueue.ResourceType] = routingQueue.ResourceRoutingQueue()
 	providerResources[location.ResourceType] = location.ResourceLocation()
 	providerResources[authDivision.ResourceType] = authDivision.ResourceAuthDivision()
-	providerResources["genesyscloud_journey_action_map"] = ResourceJourneyActionMap()
```

Comment:
You need to add these resources in using the `providerResources[journeyActionMap.ResourceType] = ...` syntax

Reply:
Nevermind. These tests no longer reference the journey tests.